portal/server/controllers/utilisation-report-service/utilisation-report-upload/index.js

```diff
@@ -96,8 +96,14 @@ const postUtilisationReportUpload = async (req, res) => {
         primaryNav: 'utilisation_report_upload',
       });
     }
-    // TODO FN-970 Populate month, year
```

Comment:
Should this comment remain here? It looks like we're still hard-coding the month and year values.